src/main/java/stsjorbsmod/characters/Wanderer.java

```diff
@@ -449,6 +450,12 @@ public void onVictory() {
     public void preBattlePrep() {
         super.preBattlePrep();
         snapCounter.reset();
+        HashSet<AbstractCard> allInstances = GetAllInBattleInstances.get(masterDeck.group.stream().filter(c -> c.cardID.equals(ForbiddenGrimoire.ID)).findAny().get().uuid);
```

Comment:
I don't think making it a Wanderer character inherent is the way to go, here; we've already gotten feedback about folks hoping for cards/etc to work if forced into other characters, and while I don't think we should bend over backwards to make that work perfect, I also would prefer to avoid regressing existing behavior that worked perfectly fine for those cases

Reply:
(my preference would be to add an `onEntomb()` hook to either `CustomJorbsModCard` or a new subscriber interface, call it from `EntombedPatch.AbstractPlayer_preBattlePrep`, and implement that on the `ForbiddenGrimoire` card to implement the marker power)